src/03-modality-agnostic-files.md

```diff
@@ -251,6 +251,71 @@ to date of birth.
 }
 ```
 
+## Samples file
+
+Template:
+
+```Text
+samples.tsv
+samples.json
+```
+
+The purpose of this RECOMMENDED file is to describe properties of samples
+when the `sample-<label>` key/value pair is used in filenames.
+If this file exists, it MUST contain the three following columns:
+
+-   `sample_id`: MUST consist of `sample-<label>` values identifying one row
+    for each sample
+
+-   `participant_id`: MUST consist of `sub-<label>`
+
+-   `sample_type`: MUST consist of sample type values (for example: tissue,
+    primary cell or cell-free sample) from [ENCODE Biosample Type](https://www.encodeproject.org/profiles/biosample_type)
```

Comment:
This may be overly restrictive. We do have the concept of TermURL, that allows researchers to associate a column with any ontology.

No strong opinion, here, though. If we want to require this TermURL for now, it can be loosened in the future to permit other ontologies.

Reply:
For Apps down the line, it seems easier to restrict the possible values.

I have question about validation though:
1. If the value MUST be from a specific ontology, does it mean that we have to plan a validation for it?
2. And if the value is REQUIRED but the ontology is not, what would we validate?

Reply:
The current set of Encode Biosample is fairly small:
`"cell line", "in vitro differentiated cells", "primary cell", "cell-free sample", "cloning host", "tissue", "whole organisms", "organoid", "technical sample"`. I could list them all in the description as accepted values if we go with a fixed ontology.

As for TermURL, I understand it the same way as @satra, as a link to the definition of what `sample_type` means rather than the link to the ontology itself. If we want to point to one of the Ontobee definition, I would go with `Biospecimen` (http://www.ontobee.org/ontology/NCIT?iri=http://purl.obolibrary.org/obo/NCIT_C70699) rather than `TissueSample` (http://purl.obolibrary.org/obo/NCIT_C19697). However, it feels strange to have a `TermURL` linking to a different ontology than the one we wish to use for the values.

Reply:
You're right about TermURL. Apologies for the confusion. I would probably go with a fixed set, and we can say that these are taken from Encode Biosample.